src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -2972,7 +3023,7 @@ private bool HasExplicitArrayConversion(TypeSymbol source, TypeSymbol destinatio
 
                 foreach (var iface in this.corLibrary.GetDeclaredSpecialType(SpecialType.System_Array).AllInterfacesWithDefinitionUseSiteDiagnostics(ref useSiteDiagnostics))
                 {
-                    if (HasIdentityConversion(iface, source))
+                    if (HasIdentityConversionInternal(iface, source, includeNullability: false))
```

Comment:
>includeNullability: false [](start = 69, length = 25)

Could you please elaborate why we unconditionally ignore nullability here?

Reply:
Removed `includeNullability: false`.

---
In reply to: [169817887](https://github.com/dotnet/roslyn/pull/24935#discussion_r169817887) [](ancestors = 169817887)